packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -4933,7 +5199,66 @@ Generated by [AVA](https://ava.li).
           origin: config␊
           package: ./plugin.js`
 
-## Provided --node-path version produces a node version warning if the build is being run in buildbot with local plugins
+## Provided --node-path version has no effect for plugin executions when the feature flag plugins_system_node_version is set
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags:␊
+        - buildbot_build_plugins_system_node_version␊
+      mode: buildbot␊
+      nodePath: /external/path␊
+      repositoryRoot: packages/build/tests/plugins/fixtures/engines␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      packages/build/tests/plugins/fixtures/engines␊
+    ␊
+    > Config file␊
+      packages/build/tests/plugins/fixtures/engines/netlify.toml␊
+    ␊
+    > Resolved config␊
+      build:␊
+        publish: packages/build/tests/plugins/fixtures/engines␊
+        publishOrigin: default␊
+      functions:␊
+        '*': {}␊
+      plugins:␊
+        - inputs: {}␊
+          origin: config␊
+          package: ./plugin.js␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    > Loading plugins␊
+       - ./plugin.js@1.0.0 from netlify.toml␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. onPreBuild command from ./plugin.js                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    ␊
+    (./plugin.js onPreBuild completed in 1ms)␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build Complete                                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    (Netlify Build completed in 1ms)`
+
+## Provided --node-path version produces a node version warning if the build is being run in buildbot with local plugins and the plugins_system_node_version ff is set to false
```

Comment:
Validate that we actually get the warning logs for incompatible Node versions if the FF is disabled.